internal/http/services/owncloud/ocs/conversions/permissions.go

```diff
@@ -39,21 +39,30 @@ const (
 	PermissionDelete
 	// PermissionShare grants share permissions on a resource
 	PermissionShare
+	// PermissionDeny grants permissions to deny access on a resource
+	// The recipient of the resource will then have PermissionNone.
+	PermissionDeny
+	// PermissionNone grants no permissions on a resource
+	PermissionNone
+	// PermissionMax is to be used within value range checks
+	PermissionMax Permissions = (1 << (iota - 1)) - 1
 	// PermissionAll grants all permissions on a resource
-	PermissionAll Permissions = (1 << (iota - 1)) - 1
+	PermissionAll = PermissionMax - PermissionNone
+	// PermissionMin is to be used within value range checks
+	PermissionMin = PermissionRead
```

Comment:
The naming is confusing. The minimal permission is `PermissionNone`. I would opt removing this completely

Reply:
This is not about the minimal permissions, as far as I understand @labkode and @ishank011 it is the upper maximum for the bitmask which is used to validate possible input values. Same for `PermissionMax`

Reply:
suggest to use `PermissionMinInput`

Reply:
Ok ja. Much better now. :+1: